packages/roosterjs-editor-dom/lib/metadata/validate.ts

```diff
@@ -8,7 +8,7 @@ import { Definition, DefinitionType } from 'roosterjs-editor-types';
  */
 export default function validate<T>(input: any, def: Definition<T>): input is T {
     let result = false;
-    if (def.isOptional && typeof input === 'undefined') {
+    if ((def.isOptional && typeof input === 'undefined') || (def.allowNull && !input)) {
```

Comment:
this is not right. If input is empty string or 0, it will also return true.

Need to check def.allowNull && input === null